### PR TITLE
Vagrant up transporter

### PR DESCRIPTION
**Pull request by franklinwise**
Enables one to just type:

```
> vagrant up
```

and have a fully configured VM with Mongo, ElasticSearch and Compose, with a ready to run demo.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+")
+
+VAGRANT_VERSION = 2
+Vagrant.configure(VAGRANT_VERSION) do |config|
+	config.vm.provider "vmware_fusion" do |v|
+		v.vmx["memsize"] = "2048"
+	end
+	config.vm.provider "virtualbox" do |v|
+		v.memory = 2048
+	end
```
Comment (nstott) [inline Vagrantfile]: I like having both fusion and virtualbox here, nice

**Comment by nstott** [comment]
I like this a lot
I'm not familiar with ansible, 
is there a chance of the dot ansible files conflicting with some other system parts?  should those provisioning files be kept in a subdir?  (I'm not sure the idiom with these)

**Comment by nstott** [comment]
lgtm
thanks!

**Comment by franklinwise** [comment]
Thank you!

**Pull request merged**